src/glowCookies.js

```diff
@@ -235,55 +235,76 @@ class GlowCookies {
+        border: obj.border || 'border',
+        position: obj.position || 'left',
+        hideAfterClick: obj.hideAfterClick || false,
+        bannerStyle: obj.style || 2
+      }
 
-    this.tracking = {
-      AnalyticsCode: obj.analytics || undefined,
-      FacebookPixelCode: obj.facebookPixel || undefined,
-      HotjarTrackingCode: obj.hotjar || undefined,
-      customScript: obj.customScript || undefined
-    }
+      this.tracking = {
+        AnalyticsCode: obj.analytics || undefined,
+        FacebookPixelCode: obj.facebookPixel || undefined,
+        HotjarTrackingCode: obj.hotjar || undefined,
+        customScript: obj.customScript || undefined
+      }
 
-    this.banner = {
-      description: obj.bannerDescription || lang.bannerDescription,
-      linkText: obj.bannerLinkText || lang.bannerLinkText,
-      link: obj.policyLink || '#link',
-      background: obj.bannerBackground || '#fff',
-      color: obj.bannerColor || '#1d2e38',
-      heading: obj.bannerHeading !== 'none' ? obj.bannerHeading || lang.bannerHeading : '',
-      acceptBtn: {
-        text: obj.acceptBtnText || lang.acceptBtnText,
-        background: obj.acceptBtnBackground || '#253b48',
-        color: obj.acceptBtnColor || '#fff'
-      },
-      rejectBtn: {
-        text: obj.rejectBtnText || lang.rejectBtnText,
-        background: obj.rejectBtnBackground || '#E8E8E8',
-        color: obj.rejectBtnColor || '#636363'
-      },
-      manageCookies: {
-        color: obj.manageColor || '#1d2e38',
-        background: obj.manageBackground || '#fff',
-        text: obj.manageText || lang.manageText,
+      this.banner = {
+        description: obj.bannerDescription || lang.bannerDescription,
+        linkText: obj.bannerLinkText || lang.bannerLinkText,
+        link: obj.policyLink || '#link',
+        background: obj.bannerBackground || '#fff',
+        color: obj.bannerColor || '#1d2e38',
+        heading: obj.bannerHeading !== 'none' ? obj.bannerHeading || lang.bannerHeading : '',
+        acceptBtn: {
+          text: obj.acceptBtnText || lang.acceptBtnText,
+          background: obj.acceptBtnBackground || '#253b48',
+          color: obj.acceptBtnColor || '#fff'
+        },
+        rejectBtn: {
+          text: obj.rejectBtnText || lang.rejectBtnText,
+          background: obj.rejectBtnBackground || '#E8E8E8',
+          color: obj.rejectBtnColor || '#636363'
+        },
+        manageCookies: {
+          color: obj.manageColor || '#1d2e38',
+          background: obj.manageBackground || '#fff',
+          text: obj.manageText || lang.manageText,
+        }
       }
-    }
 
-    // Draw banner
-    window.addEventListener('load', () => { this.render() })
+      // Draw banner
+      window.addEventListener('load', () => { this.render() })
+    }
   }
 }
 
 class LanguagesGC {
-  constructor(code) {
+  constructor(code, defaultLang) {
     this.init()
-    let lang = this.arrLang[code] || this.arrLang['en']
+    var lang = null;
```

Comment:
Let's use 'let' to maintain consistency with the rest of the project.
```suggestion
    let lang = null;
```